cmd/agentctl/client/http_client_go113.go

```diff
@@ -0,0 +1,34 @@
+//  Copyright (c) 2019 Cisco and/or its affiliates.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at:
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+
+// +build go1.13
+
+package client
+
+import (
+	"net/http"
+)
+
+// HTTPClient returns configured HTTP client.
+func (c *Client) HTTPClient() *http.Client {
```

Comment:
I think, the `HTTPClient` method should be defined once next to other `Client`'s methods. And these two ways of _cloning_ of transport for different go versions should be extracted into separate function `cloneHTTPTransport` and used like: 


```go
// HTTPClient returns configured HTTP client.
func (c *Client) HTTPClient() *http.Client {
	if c.httpClient == nil {
		tr := cloneHTTPTransport()
		tr.TLSClientConfig = c.httpTLS

		c.httpClient = &http.Client{
			Transport: tr,
		}
	}
	return c.httpClient
}
```

This way you minimize the redundant code while making it more clear what part is handled differently.